src/dune_config/config.mli

```diff
@@ -54,6 +54,10 @@ val threaded_console : Toggle.t t
 (** The number of frames per second for the threaded console. *)
 val threaded_console_frames_per_second : [ `Default | `Custom of int ] t
 
+(** The console width that dune will fall back on if it cannot determine the
+    console width. *)
+val console_width_fallback : int t
```

Comment:
Instead of calling this option the "fallback", make it:

```
val console_width : int option t
```

And if it's set, always use it instead of asking for the terminal size.